src/modules/app/scanner.js

```diff
@@ -4,7 +4,7 @@ import omit from 'lodash/omit';
 
```

Comment:
Planning to move the whole contents og scanner.js into scannerWrapper.js. Since the only place its used is inside the wrapper. I need to find a reasonable way of testing the contents of scanner.js Anyway. So my best bet is to tweak it a little bit and place it inside HoC scannerWrapper.js